ci/ray_ci/automation/docker_tags_lib.py

```diff
@@ -9,11 +9,125 @@
 import requests
 
 from ci.ray_ci.utils import logger
+from ci.ray_ci.builder_container import DEFAULT_ARCHITECTURE, DEFAULT_PYTHON_VERSION
+from ci.ray_ci.docker_container import (
+    GPU_PLATFORM,
+    PYTHON_VERSIONS_RAY,
+    PYTHON_VERSIONS_RAY_ML,
+    PLATFORMS_RAY,
+    PLATFORMS_RAY_ML,
+    ARCHITECTURES_RAY,
+    ARCHITECTURES_RAY_ML,
+)
 
 bazel_workspace_dir = os.environ.get("BUILD_WORKSPACE_DIRECTORY", "")
 SHA_LENGTH = 6
 
 
+def _check_python_version(python_version: str, ray_type: str) -> None:
+    if ray_type == "ray" and python_version not in PYTHON_VERSIONS_RAY:
+        raise ValueError(
+            f"Python version {python_version} not supported for ray image."
+        )
+    if ray_type == "ray-ml" and python_version not in PYTHON_VERSIONS_RAY_ML:
+        raise ValueError(
+            f"Python version {python_version} not supported for ray-ml image."
+        )
+
+
+def _check_platform(platform: str, ray_type: str) -> None:
+    if ray_type == "ray" and platform not in PLATFORMS_RAY:
```

Comment:
nit, put the word 'ray' and 'ray-ml' into constants too

Reply:
done. I added an enum class for it in `docker_container` and use it across the files